cmd/control/control.go

```diff
@@ -172,6 +173,12 @@ func newStartCommand() *cobra.Command {
 		"",
 		"[optional] JSON string of map from chain id to its upgrade file contents",
 	)
+	cmd.PersistentFlags().StringVar(
```

Comment:
Looks like the flags part and the map unmarshal part  are always the same? Maybe we could put them into functions?

Reply:
Would rather prefer another PR with changes for all params https://github.com/ava-labs/avalanche-network-runner/issues/352